packages/wdio-browserstack-service/src/launcher.js

```diff
@@ -0,0 +1,53 @@
+import BrowserstackLocalLauncher from 'browserstack-local';
+
+export default class BrowserstackLauncherService {
+    onPrepare(config, capabilities) {
+        if (!config.browserstackLocal) {
+            return Promise.resolve('Browserstack Local is off');
```

Comment:
just return with a log message, we don't do anything with the promise here.

Reply:
Done.